Dockerfile

```diff
@@ -8,24 +8,25 @@ WORKDIR /workspace
 COPY go.mod go.mod
 COPY go.sum go.sum
 
+# Download dependencies
+RUN go mod download
+
 # Copy the go source
 COPY cmd/main.go cmd/main.go
 COPY api/ api/
 COPY internal/controller/ internal/controller/
 COPY pkg/ pkg/
 COPY version/ version/
-COPY vendor/ vendor/
 
 # Build
 # the GOARCH has not a default value to allow the binary be built according to the host where the command
 # was called. For example, if we call make docker-build in a local env which has the Apple Silicon M1 SO
 # the docker BUILDPLATFORM arg will be linux/arm64 when for Apple x86 it will be linux/amd64. Therefore,
 # by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
 RUN CGO_ENABLED=0 \
-    GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} \
+    GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} GO111MODULE=on \
```

Comment:
The `GO111MODULE` environment variable is [on `auto` by default](https://go.dev/blog/go116-module-changes) from Go `1.16`. Therefore, adding this variable is not needed.

Reply:
removed